src/main/java/mate/academy/springbootintro/dto/cartitem/CreateCartItemRequestDto.java

```diff
@@ -0,0 +1,10 @@
+package mate.academy.springbootintro.dto.cartitem;
+
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.Positive;
+
+public record CreateCartItemRequestDto(
+        @NotBlank @Positive Long bookId,
```

Comment:
NotBlank is used only for String
```suggestion
        @NotNull @Positive Long bookId,
```